src/Raven.Client/Documents/Session/Tokens/OrderByToken.cs

```diff
@@ -121,5 +121,18 @@ public override void WriteTo(StringBuilder writer)
             if (_descending) // we only add this if we have to, ASC is the default and reads nicer
                 writer.Append(" desc");
         }
+        
+        public OrderByToken AddAlias(string alias)
+        {
+            if (_fieldName == "id()")
```

Comment:
Please use a constant instead of hard coded string. It's available here - `Constants.Indexing.Fields.DocumentIdFieldName`

Reply:
If you see hard coded "id()" in other places then please update that as well

Reply:
Fixed